platform/src/main/java/org/hillview/utils/TestTables.java

```diff
@@ -65,6 +64,45 @@ public static Table testRepTable() {
     }
 
     /**
+     * Can be used for testing large tables with strings.
+     * @param size Number of rows in the table
+     * @param others Array of options in the "Name" column
+     * @param count Number of occurrences of the 'test' string.
+     * @param test The
```

Comment:
incomplete comment.
Do not feel the need to document things which are obvious. But these are good comments.